src/main/java/com/group12/LaunchInterceptorConditionCollection.java

```diff
@@ -68,6 +70,42 @@ public boolean LIC1(List<Point> points, double radius1) throws IllegalArgumentEx
         return false;
     }
 
+    /**
+    * LIC #2 checks if the given angle which is constructed from the 3 given data points is not in the range of Pi's deviations (exclusive).
+    *
+    * @param points list of radar echods ({@link Point})
+    * @param epsilon the deviation from Pi
+    * @return true if the angle is smaller than Pi - <b>epsilon</b> or greater than Pi + <b>epsilon</b>
+    * @throws IllegalArgumentException is thrown if <b>points</b> is null, <b>points</b> does not contain 3 elements, 
+    *                                  if the first or last element of <b>points</b> coincides with the second element of <b>points</b>,
+    *                                  if 0 <= <b>epsilon</b> < Pi does not hold.
+    */
+    public boolean LIC2(List<Point> points, double epsilon) throws IllegalArgumentException{
+        if (points == null) {
+            throw new IllegalArgumentException("Points list cannot be null");
+        }
+        if (points.size() < 3) {
+            throw new IllegalArgumentException("There has to be atleast three points");        
+        }
+        if(doubleCompare(epsilon, 0) == -1 || doubleCompare(epsilon,PI) >= 0){
+            throw new IllegalArgumentException("Epsilon must hold these conditions: 0 <= Epsilon < Pi");
+        }
+        for( int i = 0; i<points.size() -2; i += 3){
+            if((points.get(i).getX() == points.get(i+1).getX()) && (points.get(i).getY() == points.get(i+1).getY()) || 
+            ((points.get(i+2).getX() == points.get(i+1).getX()) && (points.get(i+2).getY() == points.get(i+1).getY()))
+            ){
+                continue;
+            }
+            double angle = angleFromThreePoints(points.get(i+1), points.get(i), points.get(i+2));
+            if(doubleCompare(angle, PI - epsilon) == -1 || doubleCompare(angle, PI - epsilon) == 1 ){
```

Comment:
Should be:
```java
if(doubleCompare(angle, PI - epsilon) == -1 || doubleCompare(angle, PI + epsilon) == 1 )
```
I guess you can pass the test with this code means you did not cover this testcase. Better to add this test.